9/collection-meta.yaml

```diff
@@ -134,6 +134,10 @@ collections:
     maintainers:
       - Shilpi-J
     repository: https://github.com/ansible-collections/ibm.spectrum_virtualize
+  ibm.storage_virtualize:
+    maintainers:
+      - Shilpi-J
```

Comment:
Please change the name to Sumit Gupta (sumitguptaibm) as maintainer has changed.

Reply:
```suggestion
      - sumitguptaibm
```